src/core/views/components/iframe.js

```diff
@@ -20,8 +21,32 @@ class Iframe extends Component {
     this.iframeRef = node
   }
 
-  handleLoad() {
-    this.props.onLoad && this.props.onLoad(this.iframeRef)
+  async handleLoad() {
+    this.props.onLoad(this.iframeRef)
+    await this.triggerIframeRedraw()
```

Comment:
won't `this.forceUpdate()` just work for that?

Reply:
It's about asking _**the browser**_ to redraw the iframe, rather than asking react to rerender